docs/general/server/transcoding.md

```diff
@@ -20,9 +20,9 @@ This option will manually set the number of threads to use when transcoding. If
 
 ## Types of Transcoding
 
-There are three types of transcoding. The type being used will be listed in the dashboard when playing a file. They are ordered below from lowest to highest load on the server:
+There are four types of playback; three of which involve transcoding.  The type being used will be listed in the dashboard when playing a file. They are ordered below from lowest to highest load on the server:
 
-- Direct Play: Delivers the file with no modifications. Almost no additional load on the server.
+- Direct Play: Delivers the file without transcoding.  There is no modification to the file and almost no additional load on the server.
```

Comment:
```suggestion
- Direct Play: Delivers the file without transcoding. There is no modification to the file and almost no additional load on the server.
```

Reply:
Ha!  I noticed the space after I submitted and that it was inconsistent with this new-age, single space after a sentence, format the kids are using these days.  I'd have updated it previously, but this is my first fork, commit, and pull request in my life.  I wanted to take it slow and not risk screwing things up further in trying to make the update!  Many thanks!

Reply:
It's still wrong, no?